pyworkflow/plugin.py

```diff
@@ -588,3 +603,70 @@ def getHomePage(self):
     def getKeywords(self):
         return self._metadata.get('Keywords', '')
 
+class TemplateDirData():
+    def __init__(self, plugin):
+        self.path = self.getPath(plugin)
+        self.fileList = self.fillFileList()
+
+    def getPath(self, plugin):
+        path = os.path.join(os.path.dirname(plugin.__file__), "templates")
+        if os.path.exists(path):
+            return path
+        else:
+            return ""
+
+    def fillFileList(self):
+        fileList = []
+        if self.path:
+            for file in glob.glob1(self.path, "*.json.template"):
+                fileList.append(file)
+            return fileList
+
+
+class Template():
+    def __init__(self, pluginName, tempPath, description="Description not provided"):
+        self.pluginName = pluginName
+        self.templateName = os.path.basename(tempPath).replace(".json.template", "")
+        self.templateDir = os.path.abspath(tempPath)
```

Comment:
I'd rename this, Dir suffix is missleading, what about temnplateFileName or templatePath?